src/theory/bv/theory_bv_rewriter.h

```diff
@@ -50,8 +50,9 @@ class TheoryBVRewriter : public TheoryRewriter
 
  private:
   static RewriteResponse IdentityRewrite(TNode node, bool prerewrite = false);
-  static RewriteResponse UndefinedRewrite(TNode node, bool prerewrite = false); 
-  
+  static RewriteResponse UndefinedRewrite(TNode node, bool prerewrite = false);
```

Comment:
What's this for? What is it used for?

Reply:
That's unused and was probably used to initialize `d_rewriteTable` to indicate that a rewrite for a kind was not implemented. I can remove it if you want.